rtgui/filebrowser.h

```diff
@@ -181,12 +181,13 @@ class FileBrowser  : public ThumbBrowserBase,
         return tbl ? tbl->isInTabMode() : false;
     }
 
-    void openNextImage ();
-    void openPrevImage ();
+    void openNextImage();
+    void openPrevImage();
+    void selectImage(Glib::ustring fname);
+
     void copyProfile ();
     void pasteProfile ();
     void partPasteProfile ();
-    void selectImage (Glib::ustring fname);
     void openNextPreviousEditorImage (Glib::ustring fname, eRTNav eNextPrevious);
```

Comment:
@Floessie 
...then this should be `void openNextPreviousEditorImage (const Glib::ustring fname, eRTNav eNextPrevious);` too